actions/search.py

```diff
@@ -55,7 +58,11 @@ def simple_search(self):
             logger.error(e.message)
             sys.exit(2)
 
-        self._pp_exit(result)
+        if self._return_object:
+            return True, result
```

Comment:
Per the wrappers functionality I'm setting `action_status` to `True` explicitly. https://github.com/StackStorm/st2/blob/master/st2common/st2common/runners/python_action_wrapper.py#L159